spec/ndx-ophys-devices.extensions.yaml

```diff
@@ -1,8 +1,216 @@
 groups:
-- neurodata_type_def: TetrodeSeries
-  neurodata_type_inc: ElectricalSeries
-  doc: An extension of ElectricalSeries to include the tetrode ID for each time series.
+- neurodata_type_def: Indicator
+  neurodata_type_inc: Device
```

Comment:
An indicator is a device?

IDK if that makes much sense to allow `isinstance(some_indicator, Device) is True`

I'd go one level above in the heirarchy

Reply:
If this is planned to be the parent of the other fiber photometry extension as well then I'd split it from `Device`

Yes, they both share the manufacturer field, but schematically the semantics of the MRO should be distinct

General example: cars and pharmaceuticals both have manufacturers, but you would not say that `isinstance(ibuprofen, Car) is True` or `isinstance(my_car, Drug) is True`